configCivcraft.yml

```diff
@@ -42,7 +42,54 @@ factories:
     - Upgrade_to_Laboratory
     - Upgrade_to_Wood_Processor
     - Upgrade_to_Basic_Forge
+    - Upgrade_to_Basic_Fortification
     
+  #Reinforcement factories
+  #Basic Contraption --> Basic Fortification
+  basicFortification:
+    type: FCCUPGRADE
+    name: Basic Fortification
+    recipes:
+     - Produce_decent_solid_Miner_Component
+     - Produce_decent_shiny_Miner_Component
```

Comment:
As a courtesy what do you think about including the basic recipes so people can more easily mass produce them once they have the factory online?